salt/edx/run_ansible.sls

```diff
@@ -0,0 +1,73 @@
+{% set data_path = '/tmp/edx_config' -%}
+{% set venv_path = '/tmp/edx_config/venv' -%}
+{% set repo_path = '/tmp/edx_config/configuration' -%}
+{% set conf_file = '/tmp/edx_config/edx-sandbox.conf' -%}
+
+{% set playbooks = salt.pillar.get('edx:playbooks', salt.environ.get('EDX_PLAYBOOKS', ['edx-east/common.yml']).split(',')) -%}
+
+clone_edx_configuration:
+  file.directory:
+    - name: {{ repo_path }}
+    - makedirs: True
+  git.latest:
+    - name: {{ salt.pillar.get('edx:config:repo', 'https://github.com/edx/configuration.git') }}
+    - rev: {{ salt.pillar.get('edx:config:branch', 'named-release/dogwood.3') }}
```

Comment:
Shouldn't we change this to eucalyptus? Wonder if it's easier going forward to define the openedx version variable that we want to install and just use it everywhere.